plugins/acpi-dmar/meson.build

```diff
@@ -1,4 +1,5 @@
 if hsi and (host_cpu == 'x86' or host_cpu == 'x86_64')
+plugins += {meson.current_source_dir().split('/')[-1]: true}
```

Comment:
so i guess we have to do this as the directory name and plugin name are not 1:1 related.

Reply:
yup